Gordon360/Controllers/ProfilesController.cs

```diff
@@ -438,6 +439,37 @@ public async Task<ActionResult<StudentProfileViewModel>> UpdateMobilePhoneNumber
             return Ok(result);
         }
 
+        /// <summary>
+        /// Update office location (building description and room number)
+        /// </summary>
+        /// <param name="value">office location</param>
+        /// <returns></returns>
+        [HttpPut]
+        [Route("office_location/{value}")]
+        [AllowAnonymous]
```

Comment:
The [AllowAnonymous] decorators should be removed for these routes
```suggestion
 
```

Reply:
Thank you! Just removed that line